documentation/config_file_logbackxml.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: default
+title: "Puppet Server Configuration Files: logback.xml"
+canonical: "/puppetserver/latest/config_file_logbackxml.html"
+---
+
+Puppet Server’s logging is routed through the Java Virtual Machine's (JVM) [Logback library](http://logback.qos.ch/). By default, it logs to `/var/log/puppetlabs/puppetserver/puppetserver-access.log`, the log level is ‘INFO’, and Puppet Server sends nothing to `syslog`.
```

Comment:
The "main" logging actually defaults to `/var/log/puppetlabs/puppetserver/puppetserver.log`.  Inside of the `webserver` section of the `webserver.conf` file, though, you can optionally specify a configuration for HTTP request logging, e.g.:

https://github.com/puppetlabs/puppet-server/blob/206eb64b302b9ccd0db53b6ef247f86cc0a90fa4/ezbake/config/conf.d/webserver.conf#L2

That points to a *different* logback config file which controls the HTTP request logging, and by default we log the HTTP request info to `/var/log/puppetlabs/puppetserver/puppetserver-access.log`.